Urban-Fun/Cells/TimelineCell.m

```diff
@@ -6,21 +6,48 @@
 //
 
 #import "TimelineCell.h"
+#import <MapKit/MapKit.h>
 
 @implementation TimelineCell
 
 - (void)awakeFromNib {
     [super awakeFromNib];
     // Initialization code
+    [self.contentView.layer setBorderColor:(__bridge CGColorRef _Nullable)([UIColor colorNamed:@"Black"])];
```

Comment:
This can be replaced for [UIColor blackColor].CGColor
Curious, how did you find this solution?

Reply:
I found it on stackOverflow...I was trying to make the borders of each cell black so the user can tell where each cell ends.